cabal.project

```diff
@@ -31,3 +31,23 @@ tests: true
 benchmarks: true
 
 import: ./asserts.cabal
+
```

Comment:
These need to be removed before this PR is merged. So we're block on `io-sim` and `ouroboros-network` CHaP releases and integration into Consensus.

Reply:
The `ouroboros-network` release also needs to change the type of `lpGetLatestSlot` to `STM m (WithOrigin SlotNo)`. (I can't find a branch there with a commit making such a change. This orphan does, but it's not on any branch right now: https://github.com/IntersectMBO/ouroboros-network/commit/10eeb922569fa243ad062cac35b93e458f3145dd )

Reply:
Otherwise, good news: the GSM test is has passed 2.4million tests.

Reply:
> So we're block on `io-sim` and `ouroboros-network` CHaP releases and integration into Consensus.

I think it would even be fine to merge with an `io-sim` s-r-p as the new release won't change the API and is only relevant for tests. IIRC we did something similar with quickcheck-state-machine in the past. (also very minor: `io-sim` is actually released to Hackage, not to CHaP)